src/baklava/CustomNode.vue

```diff
@@ -103,17 +106,21 @@
 </template>
 
 <script lang="ts">
-import { Component } from 'vue-property-decorator';
+import { Component, Watch } from 'vue-property-decorator';
 import { Components } from '@baklavajs/plugin-renderer-vue';
 import ArrowButton from '@/inputs/ArrowButton.vue';
 import { ModelNodes } from '@/nodes/model/Types';
+import IrError from '@/app/ir/checking/irError';
+import { Getter } from 'vuex-class';
+import { Severity } from '@/app/ir/checking/severity';
 import { OverviewNodes } from '@/nodes/overview/Types';
 import { CommonNodes } from '@/nodes/common/Types';
 
-@Component({
-  components: { ArrowButton },
-})
+  @Component({
+    components: { ArrowButton },
+  })
```

Comment:
I'm being picky here but I believe this indentation is wrong. 😬

Reply:
ESLint/intellij disagree :D i'll try to make it consistent

Reply:
I did not bypass linting

Reply:
it just bypassed itself